tap-snapshots/test/lib/commands/ls.js.test.cjs

```diff
@@ -204,21 +187,6 @@ exports[`test/lib/commands/ls.js TAP ls --parseable --long with extraneous deps
 {CWD}/tap-testdir-ls-ls---parseable---long-with-extraneous-deps/node_modules/dog:dog@1.0.0
 `
 
-exports[`test/lib/commands/ls.js TAP ls --parseable --only=development > should output tree containing only development deps 1`] = `
-{CWD}/tap-testdir-ls-ls---parseable---only-development
-{CWD}/tap-testdir-ls-ls---parseable---only-development/node_modules/dev-dep
-{CWD}/tap-testdir-ls-ls---parseable---only-development/node_modules/foo
-{CWD}/tap-testdir-ls-ls---parseable---only-development/node_modules/dog
-`
-
-exports[`test/lib/commands/ls.js TAP ls --parseable --only=prod > should output tree containing only prod deps 1`] = `
-{CWD}/tap-testdir-ls-ls---parseable---only-prod
-{CWD}/tap-testdir-ls-ls---parseable---only-prod/node_modules/chai
-{CWD}/tap-testdir-ls-ls---parseable---only-prod/node_modules/optional-dep
-{CWD}/tap-testdir-ls-ls---parseable---only-prod/node_modules/prod-dep
-{CWD}/tap-testdir-ls-ls---parseable---only-prod/node_modules/prod-dep/node_modules/dog
-`
-
```

Comment:
why do we see changes to `ls` snapshots here?

Reply:
It looks like the [changes](https://github.com/npm/cli/pull/4744) @lukekarrys made to `npm ls` didn't get tests updated?

Reply:
This is a tap quirk.  Snapshots don't fail if there are *extra* snapshots that never get evaluated.  I ran `npm snap` to update my snapshots which rebuild this file too, removing the unused snapshots.

Reply:
We can land https://github.com/npm/cli/pull/4776 then rebase this PR

Reply:
> This is a tap quirk. Snapshots don't fail if there are extra snapshots that never get evaluated. I ran npm snap to update my snapshots which rebuild this file too, removing the unused snapshots.

ooohhh, right I think I saw that in the past, sounds great!